src/serlo-editor/plugins/text/components/text-editor.tsx

```diff
@@ -426,7 +426,7 @@ export function TextEditor(props: TextEditorProps) {
               <TextLeafRenderer {...props} />
             </span>
           )}
-          className="[&>[data-slate-node]]:mx-side [&_[data-slate-placeholder]]:top-0" // fixes placeholder position in safari
+          className="qa-plugin:text-editor [&>[data-slate-node]]:mx-side [&_[data-slate-placeholder]]:top-0" // fixes placeholder position in safari
```

Comment:
This colon here already seems like you have some convention in mind. Could you maybe explain in the [Best Practices page](https://github.com/serlo/documentation/wiki/Best-Practices#testing-1) why you sometimes use a colon and sometimes only dashes?